kubernetes/falco.yaml

```
@@ -0,0 +1,9 @@
+# This is not the full falco.yaml just the program_output config
+program_output:
+  enabled: true
+  keep_alive: false
+  program: "curl -d @- falcosidekick.sysdig-staging.svc.cluster.local:2801/"
```

```suggestion
  program: "curl -d @- falcosidekick.falco.svc.cluster.local:2801/"
```

I think we should add falcosidekick in same namespace than falco